### DISCUSSION TITLE
Number wall on lock screen got scrambled after recent Android 14 update

### DISCUSSION THREAD

**Discussion by n0rbertg** [NightscoutFoundation/xDrip#3119] [Bug reports]
Just had an update on Android 14 and the number wall on the lock screen got a bit "scrambled". There are these weird lines in the background instead of solid black. 

See the screenshots.

Was working fine on A14 before the recent update.

Phone: Pixel 7
Android14 build number: U1B2.230922.006

Xdrip version: 1a9d941-2023.10.06
Code: 2123100616

![Screenshot_20231008-103032](https://github.com/NightscoutFoundation/xDrip/assets/10490517/a9727cf1-4b75-4a78-8b04-dbd2e420fa81)

![Screenshot_20231008-080257](https://github.com/NightscoutFoundation/xDrip/assets/10490517/4ebaa559-9888-4d6c-9d42-4ec8770d6447)

**Comment by NoChance777** [comment]
I experience the same on Samsung S23 with Android 14
![Screenshot_20231116_111600_One UI Home](https://github.com/NightscoutFoundation/xDrip/assets/16236036/ab268515-9242-4afb-a353-a62765fe5cc1)

**Comment by Navid200** [comment]
Can you disable the setting and reenable it?
Does that reset it?

If not, can you reconfigure the settings to may be slightly move the number or to change the background image?
Does either make a difference?

**Reply by rudysroost** [reply]
Simply adjusting the size of the number to make it a bit smaller solved this for me.

**Reply by NoChance777** [reply]
What I've tried so far:
- changing position
- changing size
- changing background
- enabling/disabling

I will try your suggestions one again and let you know

**Reply by NoChance777** [reply]
So, I figured out that to make it work, I need to disable and enable this feature. The cropping problem unfortunately remains for every size of numbers. I suppose this is related to system fonts which do not fit into the area.

**Comment by eheydrick** [comment]
Experiencing this on a Pixel 7 after the December update. Disabling/Re-enabling or changing text size does not help.

**Reply by Navid200** [reply]
What version of xDrip are you using?

**Reply by spanosp** [reply]
Also experiencing same on Pixel 8 Pro
![Screenshot_20231215-205650](https://github.com/NightscoutFoundation/xDrip/assets/31421038/fcc766e4-05c8-444d-8731-87af891ecb49)

**Reply by spanosp** [reply]
c164980-2023.11.21

**Reply by Navid200** [reply]
Update to the latest Nightly please: https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.12.15

**Reply by eheydrick** [reply]
Updated to the latest nightly, 2023.12.15, and still happening.

**Comment by Navid200** [comment]
@n0rbertg 
Would you please click on "create issue from discussion" in the right pane?
Thanks

**Reply by n0rbertg** [reply]
Done
